charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
   labels:
 {{ include "datadog.labels" . | indent 4 }}
     app.kubernetes.io/component: cluster-agent
+    admission.datadoghq.com/enabled: "false"
```

Comment:
it is in the Deployment.spec.template.metadata.labels that the label needs to be added

Reply:
Thanks, forgot to remove this line after testing the label is not inherited by pods.